### DISCUSSION TITLE
Will Geist Design System Components Become Opensource?

### DISCUSSION THREAD

**Discussion by versecafe** [vercel/community#6626] [Feature Question] [answered]
### Question

Recently Vercel made the Geist design system public https://vercel.com/geist/introduction but currently all of the components are flagged as Internal and I was wondering if there are any plans to make the actual components open source.

<img width="1359" alt="image" src="https://github.com/vercel/community/assets/147033096/11444037-82f6-47c2-b4f5-b7e8814139b7">

**Comment by leerob** [comment, marked as answer]
Hey! The design system is shared publicly, but the component library is internal for Vercel. If you are looking for similar, public OSS components, check out https://ui.shadcn.com/ and https://v0.dev/ pairing!
